lib/bot_framework/message.rb

```diff
@@ -0,0 +1,92 @@
+module BotFramework
+  class Message
+    TEXT_FORMAT = {
+      plain: 'plain',
+      markdown: 'markdown',
+      xml: 'xml'
+    }
+    ATTACHMENT_LAYOUT = {
+      list: 'list',
+      carousel: 'carousel'
+    }
+
+    def initialize(session = nil)
```

Comment:
Unused method argument - `session`. If it's necessary, use `_` or `_session` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them.